intel_extension_for_transformers/llm/runtime/graph/__init__.py

```diff
@@ -217,10 +220,15 @@ def pad_token_id(self):
         return self.tokenizer.pad_token_id
 
     def __call__(self, input_ids, reinit=False, **kwargs):
+        """forward function"""
         if self.model is None:
-            self.init_from_bin(self.model_type, self.bin_file, **kwargs)
+            self.__init_from_bin(self.model_type, self.bin_file, **kwargs)
             self.generate_round = 0
         elif reinit:
             self.model.reinit()
             self.generate_round = 0
         return self.model.evaluate(input_ids.tolist())
+
+    def print_time(self):
+        """print time of each evaluation"""
+        self.model.print_time()
```

Comment:
no new line
you can set auto-format in vscode

Reply:
updated